buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,145 @@
+import kotlin.String
+import org.gradle.plugin.use.PluginDependenciesSpec
+import org.gradle.plugin.use.PluginDependencySpec
+
+/**
+ * Generated by https://github.com/jmfayard/buildSrcVersions
+ *
+ * Find which updates are available by running
+ *     `$ ./gradlew buildSrcVersions`
+ * This will only update the comments.
+ *
+ * YOU are responsible for updating manually the dependency version.
+ */
+object Versions {
+    const val com_diffplug_gradle_spotless_gradle_plugin: String = "3.24.2"
+
+    const val de_fayard_buildsrcversions_gradle_plugin: String = "0.6.1"
+
+    const val navigation_safe_args_gradle_plugin: String = "1.0.0"
+
+    const val com_android_tools_build_gradle: String = "3.5.0"
```

Comment:
No need for "String"

Reply:
I can remove it for now but since it is an auto generated file, every time you will check for dependency updates it'll just add it back.

Reply:
If it is auto-generated, let it be same.